spec/unit/cli_spec.rb

```diff
@@ -200,6 +200,25 @@
         run_command(["inspect", "--skip-files=/foo/bar,/baz", example_host])
       end
 
+      it "forwards the global --exclude option to the InspectTask" do
+        # Manually create the global exclude option. It depends on the experimental_features option,
+        # but that is evaluated when the class is loaded, not when the test is run, so it can't be
+        # stubbed.
+        if !Cli.flags[:exclude]
+          Cli.flag :exclude, negatable: false, desc: "Exclude elements matching the filter criteria"
+        end
+        expect_any_instance_of(InspectTask).to receive(:inspect_system) do |_instance, _store,
```

Comment:
Block argument expression is not on the same line as the block start.<br>Extra space before block parameter detected.